server/handlers/getAdmins.test.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-undef */
+// const request = require('supertest');
```

Comment:
are these all commented out with the aim of fixing them later? if not they shoul dbe removed

Reply:
Fixed in `jwt-authentication-login-page` branch